scripts/jtl_listener_service.py

```diff
@@ -34,8 +37,12 @@ def __init__(
         self.listener_url = f"{self.backend_url}:6000"
         # a timestamp format, others could be added...
         self.timestamp_format = timestamp_format
-        # how many records should be held before flushing to disk
+        # how many records should be held before flushing
         self.flush_size = 500
+        self.dequeuers_config = {
+            "max_threads": 50,
```

Comment:
What about putting here like 2 as a default value with a comment to increase it?